dist/getting-started/pages/settings-main.php

```diff
@@ -21,11 +21,11 @@
 	<div class="intro-wrap">
 		<div class="intro">
 			<?php if ( atomic_blocks_is_pro() ) { ?>
-				<a href="<?php echo esc_url( 'https://studiopress.com' ); ?>"><img class="atomic-logo" src="<?php echo esc_url( plugins_url( '../images/genesis-logo.svg', __FILE__ ) ); ?>" alt="<?php esc_html_e( 'Visit StudioPress', 'atomic-blocks' ); ?>" /></a>
+				<a href="<?php echo esc_url( 'https://studiopress.com' ); ?>"><img class="atomic-logo" src="<?php echo esc_url( plugins_url( '../images/genesis-pro-logo.svg', __FILE__ ) ); ?>" alt="<?php esc_html_e( 'Visit StudioPress', 'atomic-blocks' ); ?>" /></a>
 			<?php } else { ?>
 				<a href="https://atomicblocks.com"><img class="atomic-logo" src="<?php echo esc_url( plugins_url( '../images/logo.png', __FILE__ ) ); ?>" alt="<?php esc_html_e( 'Visit Atomic Blocks', 'atomic-blocks' ); ?>" /></a>
 			<?php } ?>
-			<h1><?php echo esc_html( get_admin_page_title() ); ?></h1>
+			<h1><?php esc_html_e( 'Genesis Pro Settings', 'atomic-blocks' ); ?></h1>
```

Comment:
This should use the `$atomic_blocks_plugin_name` variable set above otherwise it shows this when using standalone AB.

Reply:
On second thought, probably just need a new variable that includes "Settings" in it to avoid patching together partial strings. Easier for translators.